### PR TITLE
net-im/slack-bin: version bump

### PR DESCRIPTION
**Pull request by SpiderX**
Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

**Comment by SpiderX** [comment]
@monsieurp 
You asked in this PR https://github.com/gentoo/gentoo/pull/5713:

> why not /opt/${MY_PN}/${MY_PN}? I will edit this manually prior to merging.

Now repoman complies:

> >>> Creating Manifest for /home/spiderx/projects/gentoo/portage/net-im/slack-bin
>   KEYWORDS.dropped              1
>    net-im/slack-bin/slack-bin-3.0.0.ebuild: x86
>   ebuild.absdosym               3
>    net-im/slack-bin/slack-bin-2.8.0.ebuild: dosym '/opt/${MY_PN}/${MY_PN}'... could use relative path on line: 69
>    net-im/slack-bin/slack-bin-2.8.2.ebuild: dosym '/opt/${MY_PN}/${MY_PN}'... could use relative path on line: 69
>    net-im/slack-bin/slack-bin-3.0.0.ebuild: dosym '/opt/${MY_PN}/${MY_PN}'... could use relative path on line: 69
> 

Should I revert your last change?

File: net-im/slack-bin/slack-bin-3.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	opt/slack/libCallsCore.so"
+
+S="${WORKDIR}"
+
+src_install() {
+	insinto /usr/share/pixmaps
+	doins usr/share/pixmaps/${MY_PN}.png
+
+	newicon -s 512 usr/share/pixmaps/${MY_PN}.png ${MY_PN}.png
+	domenu usr/share/applications/${MY_PN}.desktop
```
Comment (SoapGentoo) [inline net-im/slack-bin/slack-bin-3.0.0.ebuild]: throw out `eutils`, these functions are in `desktop.eclass` now

File: net-im/slack-bin/slack-bin-3.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+	insinto /opt/${MY_PN}
+	doins -r usr/lib/${MY_PN}/.
+	fperms +x /opt/${MY_PN}/${MY_PN}
+	dosym /opt/${MY_PN}/${MY_PN} usr/bin/${MY_PN}
```
Comment (SoapGentoo) [inline net-im/slack-bin/slack-bin-3.0.0.ebuild]: make these relative: `dosym ../../opt/${MY_PN}/${MY_PN} /usr/bin/${MY_PN}` (thats what the repoman complaining is about)

File: net-im/slack-bin/slack-bin-3.0.0.ebuild (hunk 1 above)
Comment (SpiderX) [reply net-im/slack-bin/slack-bin-3.0.0.ebuild]: Removed eutils, added desktop.

File: net-im/slack-bin/slack-bin-3.0.0.ebuild (hunk 2 above)
Comment (SpiderX) [reply net-im/slack-bin/slack-bin-3.0.0.ebuild]: Applied.

**Comment by kensington** [comment]
Thanks! Sorry about the slow merge.

**Pull request closed without merging**